### ISSUE TITLE
mistake about setting joint acceleration 

### ISSUE THREAD

**Issue by jacknlliu** [IFL-CAMP/iiwa_stack#203]
https://github.com/IFL-CAMP/iiwa_stack/blob/4bdc91472497e0964d3d29990b8c5657ab153083/iiwa_ros/src/service/path_parameters.cpp#L65

it should be 
```
config_.request.joint_relative_acceleration = joint_relative_acceleration;
```
right?

**Comment by SalvoVirga** [comment]
You are very correct!
Feel free to open a PR for this and thanks for catching it!

**Comment by SalvoVirga** [comment]
fixed with #204
